DependencyInjection/Configuration.php

```diff
@@ -25,6 +25,18 @@ public function getConfigTreeBuilder()
               ->defaultValue(null)
               ->info('Custom logo URL')
             ->end()
+            ->scalarNode('xapi_endpoint')
+              ->defaultValue(null)
+              ->info('LRS endpoint for xAPI')
+            ->end()
+            ->scalarNode('xapi_auth')
+              ->defaultValue(null)
+              ->info('LRS auth token for xAPI')
+            ->end()
+            ->scalarNode('access_control_class')
```

Comment:
No use a `scalarNode`. Use an `enumNode` with values

https://symfony.com/doc/current/components/config/definition.html#enum-nodes

Reply:
Paella force a default access_control_class if param not exist or it's empty